app.js

```diff
@@ -51,10 +54,19 @@ app.get('/api/credentials', (req, res, next) => {
     if (err) {
       next(err);
     } else {
-      res.json({
-        token,
```

Comment:
```js
res.json({
  token,
  serviceUrl,
  accessToken,
});
```

Reply:
need to pass either/or not both of them.